batcher/aligned/generate_proof_and_send.sh

```diff
@@ -44,6 +44,7 @@ cmd=(
     --rpc_url "$RPC"
     --batcher_url "$BATCHER_CONN"
     --network "$NETWORK"
+    --max_fee 10000000000000000000000000000000
```

Comment:
Why you set a too high max_fee?

Reply:
It was for testing with spamoor. Restored in [#2701272](https://github.com/yetanotherco/aligned_layer/pull/1426/commits/27012725d548d0257c956da64976b40af942709c)